src/dialects/db2/query-generator.js

```diff
@@ -185,15 +185,6 @@ export class Db2QueryGenerator extends AbstractQueryGenerator {
     return 'SELECT TABNAME AS "tableName", TRIM(TABSCHEMA) AS "tableSchema" FROM SYSCAT.TABLES WHERE TABSCHEMA = USER AND TYPE = \'T\' ORDER BY TABSCHEMA, TABNAME';
   }
 
-  dropTableQuery(tableName) {
```

Comment:
Why do we remove this here? Is the Abstract function for this the same now?

Reply:
Yep! It was overwritten to remove 'IF EXISTS' but our minimum supported db2 version does support `IF EXISTS` here